test/e2e/apimachinery/resource_quota.go

```diff
@@ -110,7 +121,13 @@ var _ = SIGDescribe("ResourceQuota", func() {
 		framework.ExpectNoError(err)
 	})
 
-	ginkgo.It("should create a ResourceQuota and capture the life of a secret.", func() {
+	/*
+		Release: v1.16
+		Testname: ResourceQuota: Capture Secret
+		Description: Create a ResourceQuota, and then create a secret, the ResourceQuota MUST capture the
+		             secret creation. When delete the secret, the ResourceQuota MUST release usage.
```

Comment:
Is this test no longer flaky?  For a very long time it was.  @kubernetes/sig-api-machinery-api-reviews am I recalling correctly?